code/modules/projectiles/projectile/energy.dm

```diff
@@ -110,7 +110,7 @@
 	pass_flags = PASSTABLE | PASSGLASS | PASSGRILLE
 	kill_count = 100
 	embed = 0
-	incinerate = 40
+//	incinerate = 40

```

Comment:
If you read the entire folder, you would've seen that it comments out the incineration function of the BFG. That said, I don't recall doing this, but an incinerate function isn't really necessary for the BFG anyways since it gibs on hit.